plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -90,6 +90,14 @@ public void testDropColumn()
         // Dropping columns are not supported by the connector
     }
 
+    @Test
+    @Override
+    public void testRenameTableAcrossSchema()
+    {
+        // Schema creation is not supported
+        // Cannot test ALTER TABLE RENAME across schemas without CREATE SCHEMA
```

Comment:
Is ALTER TABLE RENAME supported in Accumulo connector?

Reply:
I've changed a bit the test (and commented out stuff related to creating a new schema) to make sure that it does just the table renaming stuff, and it works indeed

```
        assertUpdate("ALTER TABLE " + tableName + " RENAME TO " + tableName+"_new");

        assertFalse(getQueryRunner().tableExists(getSession(), tableName));
        assertTrue(getQueryRunner().tableExists(getSession(), tableName+"_new"));
```

Reply:
I've added therefore an empty implementation in the test for the method `testRenameTableAcrossSchema`